include/measurement_kit/nettest.hpp

```diff
@@ -0,0 +1,1922 @@
+// Part of Measurement Kit <https://measurement-kit.github.io/>.
+// Measurement Kit is free software under the BSD license. See AUTHORS
+// and LICENSE for more information on the copying conditions.
+#ifndef MEASUREMENT_KIT_NETTEST_HPP
+#define MEASUREMENT_KIT_NETTEST_HPP
+
+/*-
+ *  __      __  _____ __________  _______  .___ _______    ________
+ * /  \    /  \/  _  \\______   \ \      \ |   |\      \  /  _____/
+ * \   \/\/   /  /_\  \|       _/ /   |   \|   |/   |   \/   \  ___
+ *  \        /    |    \    |   \/    |    \   /    |    \    \_\  \
+ *   \__/\  /\____|__  /____|_  /\____|__  /___\____|__  /\______  /
+ *        \/         \/       \/         \/            \/        \/
+ *
+ * Autogenerated by `./script/autoapi/autoapi`. DO NOT EDIT!!!
+ */
```

Comment:
Added big warning at top. Coupled with new integration test, that's probably enough to make it difficult for people to make changes to the header rather than to the template and not notice it.